test/testUtils/ConsignmentStatusesOptions.scala

```diff
@@ -266,11 +273,91 @@ object ConsignmentStatusesOptions {
       "Resume transfer"
     ),
     (
-      "confirm transfer completed",
+      "descriptive metadata entered only",
+      generateStatuses(
+        seriesCompleted ++ taCompleted ++ clientChecksCompleted ++ uploadCompleted ++ antivirusCompleted ++ checksumCompleted ++ ffidCompleted ++ descriptiveMetadataEntered ++ closureMetadataNotEntered,
+        includeDefaultStatuses = false
+      ),
+      "/additional-metadata",
+      "In Progress",
+      "Resume transfer"
+    ),
+    (
+      "descriptive metadata entered and incomplete closure metadata",
+      generateStatuses(
+        seriesCompleted ++ taCompleted ++ clientChecksCompleted ++ uploadCompleted ++ antivirusCompleted ++ checksumCompleted ++ ffidCompleted ++ descriptiveMetadataEntered ++ closureMetadataNotEntered,
```

Comment:
The description says "incomplete closure metadata" but you're using the `closureMetadataNotEntered` variable. Is that right?

Reply:
Good spot. I blame copy and paste...